config/environments/production.rb

```diff
@@ -59,16 +61,29 @@
   # config.active_job.queue_name_prefix = "project_name_snake_production"
 
   config.action_mailer.perform_caching = false
-
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = {
-    address: "smtp-relay.sendinblue.com",
-    port: 587,
-    user_name: ENV.fetch("SENDINBLUE_EMAIL", nil),
-    password: ENV.fetch("SENDINBLUE_PASSWORD", nil),
-    authentication: "login",
-    enable_starttls_auto: true
-  }
+
+  if ENV.key?("SENDINBLUE_USERNAME")
+    config.action_mailer.smtp_settings = {
+      address: "smtp-relay.sendinblue.com",
+      port: 587,
+      user_name: ENV.fetch("SENDINBLUE_EMAIL"),
+      password: ENV.fetch("SENDINBLUE_PASSWORD"),
+      authentication: "login",
+      enable_starttls_auto: true
+    }
+  elsif ENV.key?("MAILTRAP_API_TOKEN")
```

Comment:
I would have preferred the app to break on boot when both of the keys are missing, but that unfortunately breaks our CI due to the asset precompilation step leading the production env settings.